hols/oci-devops/.gitignore

```diff
@@ -0,0 +1,44 @@
+Thumbs.db
+.DS_Store
+.gradle
+build/
+target/
+out/
+.idea
+*.iml
+*.ipr
+*.iws
+.project
+.settings
+.classpath
+.factorypath
+.vscode
+.theia
+
+# Compiled class file
+*.class
+
+# Log file
+*.log
+
+# BlueJ files
+*.ctxt
+
+# Mobile Tools for Java (J2ME)
+.mtj.tmp/
+
+# Package Files #
+*.jar
+*.war
+*.nar
+*.ear
+*.zip
+*.tar.gz
+*.rar
+
+# virtual machine crash logs, see http://www.java.com/en/download/help/error_hotspot.xml
+hs_err_pid*
+
+# Terraform
```

Comment:
Maybe just add these to the top level .gitignore?

Reply:
Good point, so I made the update base on that recommendation. Also since .gitignore is needed in this lab, I updated the Readme to pull it from the root of the repo.